wenet/utils/mask.py

```diff
@@ -35,8 +35,11 @@ def subsequent_mask(
          [1, 1, 0],
          [1, 1, 1]]
     """
-    ret = torch.ones(size, size, device=device, dtype=torch.bool)
-    return torch.tril(ret, out=ret)
+    arange = torch.arange(size, device=device)
```

Comment:
Add explanation about why change it like this.

Reply:
> Add explanation about why change it like this.

Tril is not supported in onnx export. See issue: https://github.com/pytorch/pytorch/issues/34129